tests/unit_tests.cpp

```diff
@@ -46,19 +85,279 @@ TEST_F(PamModelsHillTests, compare_to_known)
     {
         double a_init = 0.5;
         double l_MTC_init = 0.0;
+
         pam_models::hill::Muscle muscle = pam_models::hill::from_json(
             JSON_TEST_CONFIG_FILE, a_init, l_MTC_init);
 
         double l_MTC = 0.05;
         double dot_l_MTC = 0.001;
         double a = 0.5;
         double l_CE = 0.03;
+
         std::tuple<double, double> f_dot_lce =
             muscle.get(l_MTC, dot_l_MTC, a, l_CE);
+
         double force = std::get<0>(f_dot_lce);
         double dot_l_CE = std::get<1>(f_dot_lce);
 
         ASSERT_NEAR(force, ref_force, 0.001);
         ASSERT_NEAR(dot_l_CE, ref_dot_l_CE, 0.001);
     }
 }
+
+TEST_F(PamModelsHillTests, CE_compare_isometric_force)
+{
```

Comment:
would be nice to add for each test a small high level comment on what the test tests

Reply:
for each test the section has a small inline comment. I additionally added a description for each testing function.